src/app_service/service/message/service.nim

```diff
@@ -260,11 +260,8 @@ QtObject:
       error "error: received `chats` array for handling messages update is empty"
       return
 
-    # Temporary commented until we provide appropriate flags on the `status-go` side to cover all sections.
-    # blocking contact deletes the chat on the `status-go` side, after unblocking it, `active` prop is still false
```

Comment:
Is the blocking issue mentioned here fixed with the status-go changes?

Reply:
This is quite old line, it's better to get some details from @saledjenic. However, i did not noticed anything unexpected with blocking

Reply:
As said there in the old implementation (from the time when that comment was added) once user is blocked we were removing chat on the status-go side, then after user gets unblocked we were adding new chat for it where `active` prop wasn't set and was false in that case. What's the state of that and is that changed since then on the status-go side I really don't know.

Reply:
This change will definitely not break blocking/unblocking, but there are several things broken in master i mentioned here:  https://github.com/status-im/status-desktop/issues/11505